config/suit/templates/nrf54h20/matter/v1/app_envelope.yaml.jinja2

```diff
@@ -84,6 +86,14 @@ SUIT_Envelope_Tagged:
     suit-current-version: {{ DEFAULT_VERSION }}
 {%- endif %}
 
+{%- if 'CONFIG_SUIT_DFU_CACHE_EXTRACT_IMAGE' not in application['config'] or application['config']['CONFIG_SUIT_DFU_CACHE_EXTRACT_IMAGE'] == '' %}
```

Comment:
You may also add candidate-verification sequence to check if the payload is present inside external memory before applying modifications to the system :slightly_smiling_face: